public_html/css/sass/game.scss

```diff
@@ -0,0 +1,76 @@
+$total-height: 100%;
+
+#game-view, #canvas-view, #color-palette-view, 
+.total-area {
+  height: $total-height;
+}
+
+#canvas-view, #color-palette-view {
+  background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.